powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthService.java

```diff
@@ -17,662 +17,1915 @@
-     *
-     * @param request Operation failed approval request.
-     * @return Operation failed approval response.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationUserActionResponse failApprovalOperation(OperationFailApprovalRequest request) throws Exception;
-
-    /**
-     * Reject operation by ID and other operation attributes.
-     * @param request Operation reject request.
-     * @return Operation reject response.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationUserActionResponse rejectOperation(OperationRejectRequest request) throws Exception;
-
-    /**
-     * Get all operation templates.
-     * @return Operation templates.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationTemplateListResponse getAllTemplates() throws Exception;
-
-    /**
-     * Get operation template detail.
-     * @param request Request with ID to fetch operation template detail for.
-     * @return Operation template detail.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationTemplateDetailResponse getTemplateDetail(OperationTemplateDetailRequest request) throws Exception;
-
-    /**
-     * Create a new operation template.
-     * @param request Parameters of a new operation template.
-     * @return Operation template detail.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationTemplateDetailResponse createOperationTemplate(OperationTemplateCreateRequest request) throws Exception;
-
-    /**
-     * Update an existing operation.
-     * @param request Parameters of an updated template with provided ID.
-     * @return Operation template detail.
-     * @throws Exception In case of a business logic error.
-     */
-    OperationTemplateDetailResponse updateOperationTemplate(OperationTemplateUpdateRequest request) throws Exception;
-
-    /**
-     * Remove operation template.
-     * @param request Operation template with provided ID.
-     * @throws Exception In case of a business logic error.
-     */
-    void removeOperationTemplate(OperationTemplateDeleteRequest request) throws Exception;
-
+@Component("powerAuthServiceImplV3")
+public class PowerAuthService {
+
+    private PowerAuthServiceConfiguration powerAuthServiceConfiguration;
+
+    private ServiceBehaviorCatalogue behavior;
+
+    private LocalizationProvider localizationProvider;
+
+    private BuildProperties buildProperties;
+
+    private final ActivationStatusConverter activationStatusConverter = new ActivationStatusConverter();
+
+    // Minimum date for SQL timestamps: 01/01/1970 @ 12:00am (UTC)
+    private static final Date MIN_TIMESTAMP = new Date(1L);
+
+    // Maximum date for SQL timestamps: 01/01/9999 @ 12:00am (UTC)
+    private static final Date MAX_TIMESTAMP = new Date(253370764800000L);
+
+    // Prepare logger
+    private static final Logger logger = LoggerFactory.getLogger(PowerAuthService.class);
```

Comment:
Let`s reorder it little bit.
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(PowerAuthService.class);
    
    // Minimum date for SQL timestamps: 01/01/1970 @ 12:00am (UTC)
    private static final Date MIN_TIMESTAMP = new Date(1L);

    // Maximum date for SQL timestamps: 01/01/9999 @ 12:00am (UTC)
    private static final Date MAX_TIMESTAMP = new Date(253370764800000L);

    private PowerAuthServiceConfiguration powerAuthServiceConfiguration;

    private ServiceBehaviorCatalogue behavior;

    private LocalizationProvider localizationProvider;

    private BuildProperties buildProperties;

    private final ActivationStatusConverter activationStatusConverter = new ActivationStatusConverter();
```